src/main/java/xyz/aadev/generitech/client/gui/power/GuiPowerStorage.java

```diff
@@ -17,25 +17,86 @@
  */
 
+import net.minecraft.client.gui.GuiButton;
 import net.minecraft.entity.player.InventoryPlayer;
 import xyz.aadev.aalib.client.gui.GuiBase;
 import xyz.aadev.generitech.Reference;
-import xyz.aadev.generitech.common.container.power.ContanierGenerator;
+import xyz.aadev.generitech.client.gui.button.ButtonSides;
+import xyz.aadev.generitech.common.container.power.ContanierPowerStorage;
+import xyz.aadev.generitech.common.network.Network;
+import xyz.aadev.generitech.common.network.messages.power.PacketSides;
 import xyz.aadev.generitech.common.tileentities.power.TileEntityPowerStorage;
 
+import java.io.IOException;
+
 public class GuiPowerStorage extends GuiBase {
 
-    public GuiPowerStorage(InventoryPlayer inventoryPlayer, TileEntityPowerStorage tileEntity) {
-        super(Reference.MOD_ID, new ContanierGenerator(inventoryPlayer, tileEntity));
+    TileEntityPowerStorage tileEntity;
+    private int[] sides;
+
+
+    public GuiPowerStorage(InventoryPlayer inventoryPlayer, TileEntityPowerStorage tileEntity, int[] sides) {
+        super(Reference.MOD_ID, new ContanierPowerStorage(inventoryPlayer, tileEntity));
+        this.sides = sides;
+        this.xSize = 176;
+        this.ySize = 166;
+        this.tileEntity = tileEntity;
     }
 
     @Override
     public void drawBG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
-// Do nothing because of not implemented
+        bindTexture("gui/upgrade/upgrade.png");
+        drawTexturedModalRect(paramInt1, paramInt2, 0, 0, this.xSize, this.ySize);
     }
 
     @Override
     public void drawFG(int paramInt1, int paramInt2, int paramInt3, int paramInt4) {
 // Do nothing because of not implemented
     }
+
+    @Override
+    public void initGui() {
+        int Forward = tileEntity.getForward().getIndex();
+        int Back = tileEntity.getForward().getOpposite().getIndex();
+        int Left = 4;
+        int Right = 5;
+        if (Forward == 3) {
+            Left = 5;
+            Right = 4;
+        }
+        if (Forward == 4) {
+            Left = 3;
+            Right = 2;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AS00117)